rero_ils/modules/users/jsonschemas/users/user-v0.0.1.json

```diff
@@ -0,0 +1,85 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "user",
+  "description": "JSON schema for a user.",
+  "additionalProperties": false,
+  "propertiesOrder": [
+    "first_name",
+    "last_name",
+    "street",
+    "postal_code",
+    "city",
+    "birth_date",
+    "username",
+    "phone",
+    "keep_history"
+  ],
+  "required": [
+    "$schema",
+    "first_name",
+    "last_name",
+    "birth_date",
+    "username"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate user records against.",
+      "type": "string",
+      "minLength": 9,
```

Comment:
Pattern validation should be nicer.
`"pattern": "^https://ils.rero.ch/schemas/users/*$"`